README.md

```diff
@@ -25,15 +26,45 @@ Also:
 - **Safety:** Smithay will target to be safe to use, because Rust.
 - **Modularity:** Smithay is not a framework, and will not be constraining. If there is a
   part you don't want to use, you should not be forced to use it.
-- **High-level:** You should be able to not have to worry about gory low-level stuff (but 
+- **High-level:** You should be able to not have to worry about gory low-level stuff (but
   Smithay won't stop you if you really want to dive into it).
 
-
 ## Anvil
 
 Like others, Smithay as a compositor library has its own sample compositor: anvil.
 
-You can run it with cargo after having cloned this repository:
+You can run it with cargo after having cloned this repository and installed the required dependencies:
+
+### Feature: Default
+
+- `libudev`
+- `libinput`
+- `libseat`
+- `libgbm`
+- `libsystemd`
+- `xwayland`
```

Comment:
I'm pretty sure that Anvil by default uses `libwayland` (C implementation of Wayland) so it has to be installed as well.

(It is possible to use pure rust version of wayland-rs that does not require libwayland, but that's another topic)

Reply:
Also we depend by default on `winit`, which probably also includes some new dependencies. (Like `libwayland-client` instead of just `libwayland-server`.)

Additionally this only seems to contain directly linked libraries, but we open a few at runtime.
Most importantly we need `libglvnd` and the `libegl` counter-parts currently.